studio/src/pages/[organizationSlug]/[namespace]/graph/[slug]/cache-operations.tsx

```diff
@@ -69,6 +92,46 @@ const CacheOperationsPage: NextPageWithLayout = () => {
     },
   });
 
+  if (plan !== "enterprise") {
+    return (
+      <EmptyState
+        icon={<InfoCircledIcon className="h-12 w-12" />}
+        title="Cache Warmer is not available"
+        description="Please upgrade to the enterprise plan to use the cache warmer."
+        actions={
+          <Button
+            onClick={() => {
+              router.push(`/${router.query.organizationSlug}/billing`);
+            }}
+          >
+            Upgrade
```

Comment:
You cannot upgrade this via billing? Does it make sense to get the user to billing in this case? They should contact sales. If contact sales is available from billing and that's intuitive, it's ok.

Reply:
Yep, we have contact us in billing.